docs/content.go

```diff
@@ -664,6 +664,48 @@ Enterprise Performance Computing (EPC)`
   with the image determined by the flags you pass.`
 	InspectExample string = `
   $ singularity inspect ubuntu.sif`
+
+	// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+	// Apps
+	// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+	AppsUse   string = `apps <image path>`
+	AppsShort string = `List available apps within a container`
+	AppsLong  string = `
+  List applications (apps) installed in a container, located at
+  /scif/apps. See http://containers-ftw.org/SCI-F/
+  
+  To access apps, use shell, exec, run, inspect with --app <appname>
+
+  The following environment variables are available to you when called 
+  from the shell inside the container. The top variables are relevant 
+  to the active app (--app <app>) and the bottom available for all 
+  apps regardless of the active app:
+
+  ACTIVE APP ENVIRONMENT:
+
+      SCIF_APPNAME       the name of the application
+      SCIF_APPROOT       the application base (/scif/apps/<app>)
+      SCIF_APPMETA       the application metadata folder
+      SCIF_APPDATA       the data base folder for active app
+        SCIF_APPINPUT    expected input folder within data base folder
```

Comment:
ident

Reply:
This is the appropriate indentation here, this was taken from `2.x`'s help text

Reply:
Yeah, the indentation does look a little strange on this line.

Nitpick: the copyright header needs to be updated in this file also.